daemon/meson.build

```diff
@@ -1,12 +1,17 @@
 # Machine Learning Agent
 # This file should be included when ml-service feature is enabled.
 nns_ml_agent_incs = include_directories('includes')
-nns_ml_agent_srcs = files('main.c', 'modules.c', 'gdbus-util.c',
-  'service-db.cc', 'pipeline-dbus-impl.cc', 'model-dbus-impl.cc')
-if get_option('enable-tizen')
-  nns_ml_agent_srcs += files('pkg-mgr.c')
+nns_ml_agent_common_srcs = files('main.c', 'modules.c', 'gdbus-util.c',
+  'pipeline-dbus-impl.cc', 'model-dbus-impl.cc')
+nns_ml_agent_service_db_srcs = files('service-db.cc')
+
+nns_ml_agent_tizen_only_srcs = files()
+if (get_option('enable-tizen'))
+  nns_ml_agent_tizen_only_srcs += files('pkg-mgr.c')
```

Comment:
how about adding pkg-mgr.c into nns_ml_agent_common_srcs?
then nns_ml_agent_tizen_only_objs at line 25 would be unnecessary.

Reply:
I am not sure what your suggestion is.
1) Just use the common_srcs/objs, which conditionally include ```pkg-mgr.c```
2) Write a dummy one for not-tizen platforms


FYI, 1 is impossible because meson does not allow to append a obj to the obj list.

Reply:
```
if (get_option('enable-tizen'))
  nns_ml_agent_common_srcs += files('pkg-mgr.c')
endif

# then, line 25~27 extracting obj from tizen_only_srcs is unnecessary (nns_ml_agent_tizen_only_objs).
```

Reply:
Understood:(